deploy/kiali/kiali_cr.yaml

```diff
@@ -796,6 +796,8 @@ spec:
 # of the URL (usually it's a fully-qualified domain name).
 # For example, "kiali.example.org".
 # When empty, Kiali will try to guess this value from HTTP headers.
+# You must populate this value if you want to enable cross-linking between Kiali
+# instances in a multi-cluster setup.
```

Comment:
just a comment - is this true if on openshift?

If the server will use the route to autodiscover the external URL, is it true to say "You must populate" this value (and the web_schema, since it has the same comment)? Or should we prefix this comment with something like, "If Kiali is deployed in a non-OpenShift cluster, [you must populate this value....blah blah blah]"

Reply:
Oh, yes.... I think we should add the prefix.
Anyway, at the moment I'm testing only with K8s... I haven't spent the time to setup two OSs :-/.

Reply:
@jmazzitelli I added the prefix... and, now, I need your re-approval because GitHub drop your previous approval.